source-code/core/app/src/errors.ts

```diff
@@ -18,3 +18,17 @@ export class ConfigPathNotFoundError extends Error {
 		this.name = "ConfigPathNotFoundError"
 	}
 }
+
+export class PluginSaveError extends Error {
+	constructor(message: string, options: ErrorOptions) {
+		super(message, options)
+		this.name = "PluginSaveError"
+	}
+}
```

Comment:
I vote for more being more explicit like `PluginLoadMessagesError`, `PluginSaveMessagesError`